so-eclipse-plugin/src/com/sohelper/datatypes/StackoverflowAnswer.java

```diff
@@ -28,6 +29,18 @@ public StackoverflowAnswer(Element element) {
 				this.isAccepted = false;
 			}
 			
+			String votedUp = element.select("span[class=vote-count-post]").text();
+			if(votedUp != null && !votedUp.isEmpty()){
+				int votedUpCount = Integer.parseInt(votedUp);
+				if(votedUpCount != 0 && votedUpCount >0){
+					this.isUpVoted = true;
```

Comment:
Just a tip - These two lines can be reduced to:

`this.isUpVoted = (votedUpCount != 0 && votedUpCount > 0);`

I find it neater.